CSS/style.css

```diff
@@ -0,0 +1,466 @@
+  z-index: 1000;
+}
+
+.burger {
+  width: 35px;
+  height: 5px;
+  background-color: #333;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+
+.show-links .bar1 {
+  transform: translate(0, 11px) rotate(-45deg);
+  background-color: #fff;
+}
+
+.show-links .bar2 {
+  opacity: 0;
+}
+
+.show-links .bar3 {
+  transform: translate(0, -11px) rotate(45deg);
+  background-color: #fff;
+}
+
+.show-links .mobile-nav {
+  transform: translateX(0);
+}
+
+.nav-btn {
+  border: 3px solid #ec5242;
+  color: #ec5242;
+  padding: 8px 16px;
+  text-align: center;
+  font-size: 1rem;
+  font-weight: 500;
+
+  /* box-shadow: #fff; */
+  display: block;
+  margin: 0 auto;
+}
+
+.nav-btn:hover {
+  background-color: #ec5242;
+  color: #272a31;
+  font-weight: 300;
+}
+
+/* hero section */
+.hero {
+  background: url(../images/hero-bg.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+}
+
+.hero-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 10rem;
+}
+
+.hero-img {
+  width: 300px;
+}
+
+.hero-subtitle {
+  border: 1px solid #fff;
+  border-radius: 5px;
+  padding: 1rem;
+  margin-top: 1rem;
+  font-size: 1rem;
+  background-color: #eee;
+}
+
+.venue {
+  color: #272a31;
+  font-weight: 900;
+  padding: 1px;
+  font-size: 2rem;
+  margin: 0;
+}
+
+.venu-info {
+  font-size: 1rem;
+}
+
+/* program section */
+.program {
+  background: url(../images/program-bg.jpg);
+}
+
+.program-title {
+  text-align: center;
+  padding-top: 1em;
+  padding-left: 1em;
+  color: #fff;
+}
+
+.underline {
+  width: 100px;
+  height: 2px;
+  margin-left: auto;
+  margin-right: auto;
+  background: #ec5242;
+  border-radius: 24px;
+  margin-top: 15px;
+}
+
+.program-cards {
+  background: transparent;
+  display: flex;
+  flex-direction: column;
+  margin-top: 2em;
+  gap: 10px;
+  width: 100%;
+}
+
+.work {
+  color: #ec5242;
+  font-size: 18px;
+}
+
+.info {
+  padding: 0.25rem;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 0.5fr 1fr 3fr;
+  align-items: center;
+  text-align: justify;
+  gap: 20px;
+  background: #b3b3b351;
+  color: #fff;
+  padding-right: 12px;
+  padding-left: 12px;
+  opacity: 0.9;
+}
+
+.card:hover {
+  border: 2px solid #ec5242;
+  transform: scale(1.02);
+  box-sizing: border-box;
+}
+
+.fa,
+.fa-solid {
+  font-size: 18px;
+}
+
+.card > p {
+  text-align: center;
+}
+
+.program-btn {
+  margin: 1rem;
+  text-align: center;
+}
+
+.program-btn a {
+  border-bottom: 1px solid #fff;
+  color: #eee;
+  font-style: italic;
+  text-align: center;
+}
+
+.program-btn a:hover {
+  border-bottom: 1px solid #ec5242;
+}
+
+/* speaker section */
+.feature-title {
+  font-size: 2.25rem;
+  color: #272a31;
+  text-align: center;
+  margin-top: 2rem;
+}
+
+/* .photo {
+  padding-top: 0px;
+} */
+
+.speaker-item {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 16px;
+  border: 1px solid #fff;
```

Comment:
- [ ] kindly consider removing this border to make your design look exactly like the one provided and that will make your work look as expect in the design

Reply:
Done